.github/workflows/build-ctlstore.yml

```diff
@@ -0,0 +1,38 @@
+name: build
+on:
+  - push
+jobs:
+  build:
+    name: Executes a full build
+    runs-on: ubuntu-latest
```

Comment:
we're not executing within a golang container anymore, which simplifies the build.